.github/workflows/test.yml

```diff
@@ -0,0 +1,30 @@
+name: TUF-on-CI repository tests
+
+on:
+  workflow_call:
+  workflow_dispatch:
+  schedule:
+    - cron: '17 4,10,16,22 * * *'
+
+permissions: {}
+
+jobs:
+  smoke-test:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Smoke test repository with a client
+        uses: jku/tuf-on-ci/actions/test-repository@issue-mgmt # FIXME repo & ref
+
+  update-issue:
+    runs-on: ubuntu-latest
+    needs: [smoke-test]
+    # During workflow_call, caller updates issue
+    if: always() && !cancelled() && github.workflow == 'test.yml'
```

Comment:
this is more complicated than the others because it's a reusable workflow that gets called:
* on schedule: in this case we need to handle the issue
* on workflow_call from publish workflow: in this case we trust the caller to handle the issue

The ugly `github.workflow == 'test.yml'` is used because the logical choice `github.event_name == 'workflow_call'` does not work because GitHub uses the original event name here (the one that triggered publish workflow)